### PR TITLE
Menu / Style Rework

### PR DESCRIPTION
**Pull request by timhow38**
Replaced the current MuiTheme as there were override issues. I've instead placed it within _app.js

I've restyled and remade the Header element. It's now called NavaMenu.

Restyled other misc items, nothing crucial.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -97,8 +96,7 @@ body {
     height: 100vh; /* Use vh as a fallback for browsers that do not support Custom Properties */
     height: calc(var(--vh, 1vh) * 100);
     max-width: 100%;
-    margin: 0 auto;
-    padding: 10px;
+    /* margin: 0 auto; */
```
Comment (RileyHunter) [inline src/App.css]: No need for commented out code in prod

File: src/components/Login.js [hunk 2]
```diff
@@ -22,6 +22,10 @@ function Login(props) {
 
     let textLabel = context.user ? "Logged in as" : "Enter Your Name";
 
-    return <TextField sx={props.sx} label={textLabel} variant="outlined" onBlur={handleUserNameChange} value={userName || context.user?.id} disabled={!!context.user}/>
+    return context.user ? (
```
Comment (RileyHunter) [inline src/components/Login.js]: This is currently hidden _and_ disabled when the user is null; should pick one or the other

File: src/components/NavaMenu.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+//import CurrentElo from './CurrentElo';
+import { useContext } from 'react';
+import AppContext from './AppContext';
+import { BottomNavigation, BottomNavigationAction } from '@mui/material';
+import HomeIcon from '@mui/icons-material/Home';
+import TimelineIcon from '@mui/icons-material/Timeline';
+import RoutingAction from './RoutingAction';
+
+
+function NavaMenu(props) {
```
Comment (RileyHunter) [inline src/components/NavaMenu.js]: What is a "Nava"? I'm familiar with "Nav" obvs. but have never heard of "Nava".

File: src/App.js [hunk 4]
```diff
@@ -1,15 +1,55 @@
...
 import { getFirestore } from 'firebase/firestore';
-import { ThemeProvider } from '@mui/material/styles';
 import CssBaseline from "@mui/material/CssBaseline";
-import muiTheme from './styles/MuiTheme';
 import AppContext from './components/AppContext';
 import config from './config';
 
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+const theme = createTheme({
```
Comment (RileyHunter) [inline src/App.js:13]: Why does this belong here vs. in it's own file?

File: src/components/BodyRouter.js [hunk 5]
```diff
@@ -31,22 +31,16 @@ function BodyRouter(props) {
...
                         </Grid>
-                        <Grid item xs={4}>
+                        <Grid item xs={12}>
                             <RoutingButton text="Hangboard Timer" routeTarget={'hangboardTimer'} />
                         </Grid>
-                        <Grid item xs={4}>
-                            <RoutingButton text="My History" routeTarget={'history'} />
-                        </Grid>
-                        <Grid item xs={4}>
-                            <RoutingButton text="Leaderboards" routeTarget={'leaderBoards'} />
```
Comment (RileyHunter) [inline src/components/BodyRouter.js:47]: Is there a way to get to the leaderboards now? I see a history.

File: src/App.css (hunk 1 above)
Comment (timhow38) [reply src/App.css]: Of course, removed commented code on new commit.

File: src/components/Login.js (hunk 2 above)
Comment (timhow38) [reply src/components/Login.js]: True, reverted in new commit.

File: src/components/NavaMenu.js (hunk 3 above)
Comment (timhow38) [reply src/components/NavaMenu.js]: Changed navaMenu to navMenu

File: src/components/BodyRouter.js (hunk 5 above)
Comment (timhow38) [reply src/components/BodyRouter.js:47]: Added leader boards to the navagation menu, its icon is a trophy.

File: src/App.js (hunk 4 above)
Comment (timhow38) [reply src/App.js:13]: I was having trouble with MuiOverrides, docos that I found through stack often said to internalize the theme in the app function, didn't see any issue to it?

**Comment by timhow38** [comment]
Closing current pull request and sending another that has these issues resolved.

**Pull request closed without merging**